app/src/behaviors/behavior_sticky_key.c

```diff
@@ -210,14 +212,6 @@ static int sticky_key_keycode_state_changed_listener(const zmk_event_t *eh) {
             continue;
         }
 
-        // If this event was queued, the timer may be triggered late or not at all.
```

Comment:
What was your reasoning for deleting this? I believe it stands as a failsafe for an edge case for a laggy timer, but I'm not 100% sure exactly when it is being used.

Reply:
`release_at` is calculated upon key press. This failsafe deactivates sticky keys while they are being held. The unit test `13-active-until-release` covers this.

I have been using this for a month now without noticing any issues. Perhaps my keymap isn't complicated enough to cause any problems?

There is certainly a way to keep the failsafe but I haven't thought much about it. Personally, I prefer not to rely on failsafes and believe that unexpected behaviors should be fixed and verified through testing. (Maybe I'm just being lazy.) Let me know what you think.